recipes/legit/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set packagename = 'legit' %}
+{% set version = "0.2.0" %}
+{% set md5 = "5df3cd402b0de8f0251a87904b5eb88c" %}
+
+package:
+  name: {{ packagename }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kennethreitz/legit
+  git_tag: v{{ version }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - legit = legit.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - clint >=0.4.1
+    - gitpython >=0.3.4
+    - six >=1.9.0
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - {{ packagename }}
+
+  commands:
+    - cd {{ environ.SRC_DIR }} && legit help
```

Comment:
On second thought, that's silly.  Your tests wouldn't be running on the other platforms.  I still think this assumption is unsafe in general, but I don't see why it isn't working here.